cephprimarystorage/cephprimarystorage/cephdriver.py

```
@@ -41,7 +41,10 @@ def create_volume(self, cmd, rsp, agent=None):
             return rsp
 
         multiple_of_MB = cmd.size % sizeunit.m == 0
-        if multiple_of_MB:
+        if cmd.format == 'qcow2':
+            vol_path = "rbd:%s" % path
+            linux.qcow2_create(vol_path, cmd.size, False)
```

Ruff (0.8.2)

`linux` may be undefined, or defined from star imports

(F405)